### ISSUE TITLE
From: Mailaddress field should be after an linebreak (or buttons on right side)

### ISSUE THREAD

**Issue by j-4** [gnome-integration-team/thunderbird-gnome#7]
Die "From: Mailaddress" field is just right to the buttons (reply, forward, delete,...) which looks ugly. Further I always search for the From address, because it appeas not really in the users view.

I would suggest to move the buttons right (like in the standard theme) or to move the "From: Address" field to the next line.

Thanks for this awesome theme. I love it!

**Comment by j-4** [comment]
see the screenshot: http://www.unet.univie.ac.at/~a0604095/Pictures/mailheader_from_address.png

**Comment by zygous** [comment]
Yep, this is an issue that's been bothering me too. Particularly annoying is if a message has been sent to more than one recipient, the Reply All button forces the rest to move further to the right. It's a big UX problem to have buttons appearing in different places - several times I've clicked the wrong one by mistake because of this.
